Library/Formula/fsevent_watch.rb

```diff
@@ -0,0 +1,16 @@
+class FseventWatch < Formula
+  homepage "https://github.com/proger/fsevent_watch"
+  url "https://github.com/proger/fsevent_watch/archive/v0.1.tar.gz"
+  sha1 "abab1599a2143dd24bcb6a78e0bc29a30337119e"
+
+  head "https://github.com/proger/fsevent_watch.git"
+
+  def install
+    bin.mkpath
+    system "make", "install", "PREFIX=#{prefix}", "CFLAGS=-DCLI_VERSION=\\\"#{version}\\\""
+  end
+
+  test do
+    system "fsevent_watch", "-V"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!